tests/regression/36-apron/100-array_octagon_keep_last_prec.c

```diff
@@ -68,7 +68,9 @@ void example1(void) {
   // Values that may be read are 1 or 2
   assert(a[z] == 1); //UNKNOWN
   assert(a[z] == 2); //UNKNOWN
-  assert(a[z] == 0); //FAIL
+
+  // Relies on option sem.int.signed_overflow assume_none
+  assert(a[z] != 0); //UNKNOWN
```

Comment:
Would this not be `FAIL` then, given that we now have ` --set sem.int.signed_overflow assume_none` ?

Reply:
Not fail, but known, since I switched from `==` to `!=`.